### PR TITLE
[Common Lisp] add proposed concepts from roman-numeral exercise.

### PR DESCRIPTION
**Pull request by verdammelt**
For #494.

### CODE REVIEW

File: languages/common-lisp/reference/exercise-concepts/roman-numerals.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Concepts of Roman Numerals
+
+This one is a bit of 'cheat' of an exericse - but the concept is real.
+
+## Concepts
+
+- Formatted Output
+
+## Commnetary
```
Comment (TheLostLambda) [inline languages/common-lisp/reference/exercise-concepts/roman-numerals.md]: This should probably be: Commentary

**Comment by TheLostLambda** [comment]
I agree that we should probably try to break up `format` into some smaller concepts along the way – it's practically a language of its own. Do we want to endeavor to create a list of those sub-concepts as part of this PR?

**Comment by verdammelt** [comment]
I think that might be a good idea - at least to get a few of the smaller contexts listed. 

What would be the main concept? Formatted Output?

**Comment by TheLostLambda** [comment]
Yeah, I think that would work, or we could just call it "Format" or "FORMAT" so that, if we do a similar thing with `loop` we could justify just "Loop" or "LOOP". As they both have some sub-concepts, They could be things like: `format-numbers`, or `loop-collection`.

On the topic of `format`, I think that it would be possible split it into:

- `format-basic`: Things like A and S, nil vs t for the stream
- `format-numbers`: Number printing options
- `format-text`: Case conversions, plurals, and new lines (I'm not a fan of the name of this one)
- `format-tables`: Justification and tables
- `format-iteration`: Iteration and jumping arguments with `~*` and friends
- `format-conditional`: Conditional printing options

That's a pretty rough draft, but let me know what you think :)

**Comment by verdammelt** [comment]
I think that these sub-concepts are a good start. And then we'll need a `format-misc` for things like `~?` and a couple others that maybe don't fit into any place good.

I'll update the doc with these ideas.

**Comment by TheLostLambda** [comment]
Those look like some good changes to me! I like `format-language` a lot better than `format-text`!

I'd be happy with merging this if you think it's ready to go!

**Comment by verdammelt** [comment]
thanks @TheLostLambda . I'll merge this - and start looking at your PRs again too so we can get those merged as well.

**Pull request merged**